.github/actions/promote-to-stable/action.yml

```diff
@@ -41,16 +41,20 @@ runs:
             continue
           fi
 
-
           echo "[DEBUG] - Build $ARCH target path."
           TARGET_PATH="rpm-plugins/${{ inputs.distrib }}/${{ inputs.stability }}/$ARCH/RPMS/${{ inputs.module }}/"
           echo "[DEBUG] - Target path: $TARGET_PATH"
 
           echo "[DEBUG] - Promoting $ARCH testing artifacts to stable."
           for ARTIFACT in ${SRC_PATHS[@]}; do
-            echo "[DEBUG] - Promoting $ARTIFACT to stable."
-            jf rt cp $ARTIFACT $TARGET_PATH --flat=true
+            echo "[DEBUG] - Downloading $ARTIFACT from TESTING."
+            jf rt download $ARTIFACT --flat
+          done
+          for ARTIFACT_DL in $(dir|grep -E "*.rpm"); do
+            echo "[DEBUG] - Promoting (upload) $ARTIFACT_DL to stable $TARGET_PATH."
+            jf rt upload "$ARTIFACT_DL" "$TARGET_PATH" --flat
           done
+          rm -f *.rpm
```

Comment:
Hi @YanMeddour,
These modifications are not part of your PR 🤔

Reply:
Yes totally my bad I tried to rebase this PR in aim to help the validation of it but it looks it goes wrong :<
@omercier and me will check about this next week